package.json

```diff
@@ -24,35 +24,35 @@
   ],
   "dependencies": {
     "hyphenate-style-name": "^1.0.0",
-    "lodash.omit": "^3.0.0",
-    "matchmedia": "^0.1.1",
-    "object-assign": "^2.0.0"
+    "matchmedia": "^0.1.2",
+    "object-assign": "^4.0.1"
   },
   "peerDependencies": {
-    "react": "^0.12.0 || ^0.13.0 || ^0.14.0"
+    "react": "^0.14.0"
```

Comment:
The sample code `React.renderComponent` doesn't work with 0.14. Since 0.14 has been out for 3 months, I thoughts it's most pragmatic to drop these old versions.